### ISSUE TITLE
Doesn't seem to work on macOS Sierra

### ISSUE THREAD

**Issue by kylebrowning** [asmallteapot/cocoapods-playgrounds#35]
```
pod playgrounds Alamofire
warning: failed to load toolchain 'swift': Could not find toolchain: swift[!] /usr/local/bin/pod install --no-repo-update

Analyzing dependencies
[!] Unable to find host target for Pods-AlamofirePlayground. Please add the host targets for the embedded targets to the Podfile.
```

**Comment by neonichu** [comment]
Yah, this only supports released versions of CocoaPods at this point.

**Comment by kylebrowning** [comment]
I am using a released version of CocoaPods.

**Comment by neonichu** [comment]
Ah sorry, I was pretty sure the embedded host target detection was only available since 1.1 beta, but I was wrong about that.

In any case, Xcode 7.3.1 is officially unsupported on Sierra and Xcode 8 support is not fully baked yet in CocoaPods, so this still has to wait for a CP 1.1 release to be feasible, unfortunately.

**Comment by jadsonlourenco** [comment]
I'm getting the same error.
**Cocoapods cocoapods-1.0.1**
**Xcode 7.3.1**
**OSX 10.11.6**
